src/bci_build/package/apache_tomcat.py

```diff
@@ -66,13 +66,13 @@ def _get_sac_supported_until(
             and os_version.is_tumbleweed
         ),
         version="%%tomcat_version%%",
-        tag_version=f"{tomcat_major}-jre{jre_version}",
+        tag_version=f"{tomcat_major}-openjdk{jre_version}",
```

Comment:
nit: rename `jre_version` to `jdk_version` since we are talking about JDK and not JRE.

Reply:
well, we still only install the JRE not the JDK (but we name the tag suffix jdk which I find a bit confusing as well)..

Reply:
Why are you proposing to change the tag suffix then if it's now more confusing?

Reply:
the package list is `f"java-{jre_version}-openjdk", f"java-{jre_version}-openjdk-headless"`, is this JRE only? I thought it was JDK

Reply:
no, the jdk is the `java-{jre_version}-openjdk-devel`